requirements-base.txt

```diff
@@ -20,49 +20,50 @@ googleapis-common-protos==1.52.0
 google-cloud-pubsub==2.2.0
 google-cloud-storage==1.35.0
 jsonschema==3.2.0
-lxml>=4.6.1,<4.7.0
+lxml==4.6.2
 maxminddb==1.5.4
-mistune>0.7,<0.9
-mmh3>=2.3.1,<2.4
+mistune==0.8.4
+mmh3==2.3.1
 parsimonious==0.8.0
-petname>=2.6,<2.7
-phonenumberslite>=8.11.0,<8.12.0
-Pillow>=8.0.0,<9.0.0  # Pillow 8.x supports Python 3.6 to 3.9
-progressbar2>=3.32,<3.33
-psycopg2-binary>=2.7.0,<2.9.0
-PyJWT>=1.5.0,<1.6.0
+petname==2.6
+phonenumberslite==8.11.3
+Pillow==8.1.0
+progressbar2==3.32.0
+psycopg2-binary==2.7.7
+PyJWT==1.5.3
 pyrsistent==0.17.3
 python-dateutil>=2.0.0,<3.0.0
```

Comment:
I will need to deal with this later as I need to bump the version on `getsentry` first.